.travis.yml

```diff
@@ -1,18 +1,5 @@
-language: python
+language: minimal
```

Comment:
why are we keeping the file?

Reply:
That was only to make this PR pass checks. If you can disable travis as part of merging, then it's not needed, Otherwise I 'd remove it in new PR after the transition.